.github/workflows/buildtest.yml

```diff
@@ -118,8 +121,17 @@ jobs:
         #####
         # Build & TEST & DEPLOY STORM
         #####
-      - name: Git clone
-        run: sudo docker exec storm git clone --branch $STORM_BRANCH $STORM_GIT_URL /opt/storm
+      - name: Git shallow clone
+        # Only clone shallow if not using master on original repo (and not for pull requests or forks)
+        if: ${{ !(github.repository_owner == 'moves-rwth' && github.ref == 'refs/heads/master') }}
+        run: |
+          # git clone cannot clone individual commits based on a sha and some other refs
+          sudo docker exec storm bash -c "mkdir /opt/storm; cd /opt/storm; git init && git remote add origin ${STORM_GIT_URL} && git fetch origin ${STORM_BRANCH} && git checkout FETCH_HEAD"
```

Comment:
Is ``--depth 1`` missing here?

Reply:
yes, I must have removed it before splitting Git clone into shallow and deep clone